addons/account/models/account_invoice.py

```diff
@@ -949,6 +949,13 @@ def _track_subtype(self, init_values):
             return 'account.mt_invoice_created'
         return super(AccountInvoice, self)._track_subtype(init_values)
 
+    @api.multi
+    def _get_groups(self):
+        self.ensure_one()
+        groups = self.env['account.tax.group']
+        for line in self.tax_line_ids:
+            groups |= line.tax_id.tax_group_id
+        return groups
```

Comment:
Hi Cedric,

Thanks for your feedback. The reason for this is that I wanted that the amount comes from a field (amount_in_group), which allows to display it easily in the report (currency symbol, currency symbol right or left of the amount, etc). If there is an easy way to do that with a float variable, I don't know about it. Maybe rendering it in the python code and returning a string?

About returning a list of tuples, it's a good idea. But the way I would implement it would be to return [(group1, any tax_line in group1), ...] or maybe [(group1's name, any tax_line in group1), ...], unless there's a better approach to my above point